src/std/rec/waveformRecord.dbd.pod

```diff
@@ -9,37 +9,452 @@
 
 =title Waveform Record (waveform)
 
-...
+The waveform record type is used to interface waveform digitizers. The record
+stores its data in arrays. The array can contain any of the supported data
+types.
 
-=head2 Record-specific Menus
+=recordtype waveform
 
-=head3 Menu waveformPOST
+=cut
 
-The MPST and APST fields use this menu to determine when to post new value
-and archive monitors respectively.
+include "menuFtype.dbd"
```

Comment:
Something, I think this line, is causing waveformRecord.h to contain duplicate definitions for `menuFtype*`.

```
In file included from ../demo.cpp:9:0:

/home/travis/.source/epics-base/include/menuFtype.h:7:5: error: redeclaration of ‘menuFtypeSTRING’

     menuFtypeSTRING                 /* STRING */,

     ^

In file included from ../demo.cpp:8:0:

/home/travis/.source/epics-base/include/waveformRecord.h:19:5: note: previous declaration ‘menuFtype menuFtypeSTRING’

     menuFtypeSTRING                 /* STRING */,

     ^
```

https://travis-ci.org/mdavidsaver/pva2pva/jobs/581491657#L3490

Reply:
After discussion here I've added a guard around the typedef generated in the menu.h and record.h files so duplicates won't cause this problem and the dbd.pod files can include any menu.dbd files they want for documentation purposes (as was done in this PR). That change is now present on the 3.15 branch.